frontend/src/app/testQueue/TestCardForm/diseaseSpecificComponents/aoeUtils.ts

```diff
@@ -0,0 +1,236 @@
+import moment from "moment";
+import React from "react";
+import _ from "lodash";
+
+import { AoeQuestionResponses } from "../TestCardFormReducer";
+import {
+  PregnancyCode,
+  respiratorySymptomDefinitions,
+  respiratorySymptomsMap,
+  SymptomDefinitionMap,
+  SyphilisHistoryCode,
+  syphilisSymptomDefinitions,
+  syphilisSymptomsMap,
+} from "../../../../patientApp/timeOfTest/constants";
+import { AOEFormOption } from "../TestCardForm.utils";
+
+export function generateAoeListenerHooks(
+  onResponseChange: (responses: AoeQuestionResponses) => void,
+  responses: AoeQuestionResponses
+) {
+  const onPregnancyChange = (pregnancyCode: PregnancyCode) => {
```

Comment:
this file is mostly just moving the AOE listener hooks into central functions / re-exporting them for use in each of the AOE form components